src/sections/LearnMoreSection.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { LearnMoreButton } from "../components/LearnMoreButton";
+
+export const LearnMoreSection = ({ data }) => {
+
+  return (
+    <div className="bg-background-lavender text-text-dark text-center flex flex-col lg:flex-row-reverse items-start relative md:items-center lg:justify-center">
+      <div className="flex-1 relative py-6 px-6 md:items-center md:max-w-[600px] lg:">
+        <h2 className="font-orelega text-xl md:text-2xl font-bold mb-4 leading-tight lg:text-4xl lg:text-left">
+          <span className="block md:hidden">
+            {data["smallHeading"]}
+          </span>
+          <span className="hidden md:block">
+            {data["heading"]}
+          </span>
+        </h2>
+        <p className="font-montserrat text-sm lg:text-left lg:text-lg">
+         
+          <span className="block md:hidden">
+            {data["smallSubheading"]}
+          </span>
+          <span className="hidden md:block">
+            {data.subheading.split("\n").map((line, index) => (
+              <React.Fragment key={index}>
+                {" "}
+                {line}
+                <br />
```

Comment:
We heard that <br> is outdated and there might be accessibility issues for it :(

//Alma & Etna